### DISCUSSION TITLE
Select from all displayed layers?

### DISCUSSION THREAD

**Discussion by NelkonParker** [xournalpp/xournalpp#4689] [Q&A]
I'm using Xournalpp 1.1.1 on Windows 10.

Is there any way to select all objects that are currently visible, regardless of which layer they are on? This would help me greatly in some document rework that I'm doing at the moment.

If not, is this a feature that is in the pipeline?

**Comment by rolandlo** [comment]
It's currently not possible, as selections are currently always attached to one single layer. 
In the nightly build there is the possibility to select objects from a different layer #4413, but not from multiple layer.

Depending on your use case writing a plugin might help. What exactly do you need to do in your document rework?

**Reply by Seanramsey7** [reply]
In my case i was drawing notes, and using layers to make modifications to the notes, without changing the original layer. It would of been very helpful to save the entire selection, multiple *shown* layers and all.

As of right now, the utility of layers is really hampered by the inability to select the entire group of items, or it requires me to manually pull ALL THE LAYERS into a single layer.
